src/fs/hardlinkDir.ts

```diff
@@ -9,24 +11,37 @@ export default async function hardlinkDir(existingDir: string, newDir: string) {
     dirs
       .map(async (relativePath: string) => {
         const existingPath = path.join(existingDir, relativePath)
-        const newPath = path.join(newDir, relativePath);
-        const stat = await fs.stat(existingPath)
+        const newPath = path.join(newDir, relativePath)
+        const stat = await fs.lstat(existingPath)
+        if (stat.isSymbolicLink()) {
```

Comment:
Yeah, sorry, I was going to bed back then already.

I usually prefer not to dedup unrelated branches of code. In this case this is of course arguable.